### PR TITLE
Sprint4 72 bundle latest build

### LINKED ISSUE #72: Weaver Components should bundle its build under "latest" directory path
Latest build should be under `dist/bundle/latest` in addition to the current `dist/bundle/vX` location.

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #72

### CODE REVIEW

File: scripts/build-wvr-components.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ const concat = require('concat');
 const package = require('../package.json');
 const majorVersion = package.version.split('.')[0];
 const dirName = `${majorVersion}x`;
-const dirPath = `dist/bundle/${dirName}`;
+const basePath = 'dist/bundle/';
+const dirPath = `${basePath}/${dirName}`;
```
Comment (jeremythuff) [inline scripts/build-wvr-components.js]: Additional `/` here

File: scripts/build-wvr-components.js [hunk 2]
```diff
@@ -17,9 +19,10 @@ const dirPath = `dist/bundle/${dirName}`;
   ];
 
   fs.ensureDir(dirPath);
+  fs.ensureDir(latestPath);
 
   await concat(files, `${dirPath}/weaver-components.js`);
+  await concat(files, `${latestPath}/weaver-components.js`);
```
Comment (jeremythuff) [inline scripts/build-wvr-components.js]: This is going to take 2x as long. We should concat once, and then copy the file, as is done for docs.

File: scripts/build-wvr-components.js [hunk 3]
```diff
@@ -17,9 +19,10 @@ const dirPath = `dist/bundle/${dirName}`;
...
   fs.copy(`${dirPath}/weaver-components.js`, "dist/docs/usage/weaver-components.js");
```
Comment (jeremythuff) [inline scripts/build-wvr-components.js]: Check if the `fs.copy` command already ensures dir. If so, then line 22 is not needed.

File: scripts/build-wvr-components.js (hunk 1 above)
Comment (rmathew1011) [reply scripts/build-wvr-components.js]: Removed from L7 and added to L9

**Pull request merged**